platforms/native/engine/jsb-videoplayer-openharmony.js

```diff
@@ -49,7 +49,7 @@ if (cc.internal.VideoPlayer) {
     cc.internal.VideoPlayerImplManager.getImpl = function (componenet) {
         return new VideoPlayerImplOpenHarmony(componenet);
     };
-    window.oh.onVideoEvent = (tag, ev, args) => {
+    jsb.onVideoEvent = (tag, ev, args) => {
```

Comment:
Will this change work in `ark runtime` mode?

Reply:
It's a bug. I'll fix it.